docker-src/webui/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:16.04
+FROM ubuntu:18.04
```

Comment:
why 18.04 and not 20.04?

Reply:
20.04 didn't work because of interaction on install of apache2, but now found a workaround